### ISSUE TITLE
With [searchable]="false", component doesn't take the focus on click and keyboard navigation doesn't work and 

### ISSUE THREAD

**Issue by kheos31** [ng-select/ng-select#2221]
**Describe the bug**
When I specify [searchable]="false", the keyboard navigation does not work on this ng-select. Moreover, this ng-select doesn't take the focus on click.

**Reproductible example**
[NgSelectDemo](https://ng-select.github.io/ng-select#/forms) Go to this demo page and executes steps to reproduce.

**To Reproduce** 
1. Go to '[NgSelectDemo](https://ng-select.github.io/ng-select#/forms)'
2. In the 'Single select with required validation' example, click on select with placeholder 'Select Age' 
3. Checks the keyboard navigation works for opened dropdown
4. In the 'Reactive form using ng-option' example, click on select with placeholder 'Select Item' 
5. Uses the keyboard navigation
6. See the keyboard navigation does not work for opened dropdown and that it is activating keyboard navigation behavior for the select manipulated on the first step

![ng-select-searchable-false-bug](https://github.com/ng-select/ng-select/assets/130769302/46e162c8-fdbd-4696-a8df-4bc55c43ac5e)

**Expected behavior**
The select with searchable false have to take the focus and the keyboard navigation should works for this select

**Desktop (please complete the following information):**
 - OS: [Windows 10]
 - Browser [chrome, firefox, safari]
 - Version [latest]

**Solution Suggest**
Even if the ng-select input is in readonly because of the [searchable]="false" option, the focus have to be set on input, and in this case, keyboard navigation will works. 
NB: Solution checked with chrome debugger and using : right click on ng-select input element into 'Elements' tab => Focus

#2229 

Thanks in advance!

**Comment by kheos31** [comment]
I think it is due to this commit : https://github.com/ng-select/ng-select/pull/2134/commits/524d8a73a2f36d8883fc4146691620165ecbb100. I checked on version 10.0.3 and all is working.

**Comment by kheos31** [comment]
Bug still effective. Also linked to https://github.com/ng-select/ng-select/issues/2230. 

MR is still open https://github.com/ng-select/ng-select/pull/2229 to correct this point, which is a blocking one to be able to upgrade. 
If Safari 16 decided to change auto-scroll behavior it is not to ng-select to manage it. It's surely a facilitator for accessibility...

**Comment by jgruttad92** [comment]
Running into this as well, after upgrading to angular 16.1.3 (which I believe uses ng-select 11 under the hood). Roughly same Desktop information on my end as the original posters.

**Comment by SERGEES** [comment]
also faced the issue...
first time appeared upgrading from 9.*.*  to 10.*.* 
using 11 also present if [searchable]="false"

**Comment by kheos31** [comment]
Any update please?

**Comment by kheos31** [comment]
Bug still present on the last version on demo page. My team cannot upgrade to Angular 16 because of this.. Reproduction in gif below :
![ng-select-searchable-false-bug](https://github.com/ng-select/ng-select/assets/130769302/a0e89a15-f51c-4964-b8d9-75670145f8c2)

Please, how can we do more to be able to upgrade?

**Comment by pavankjadda** [comment]
@kheos31 How do you know this change fixes the issue?

**Comment by pavankjadda** [comment]
@kheos31 thanks for PR. Merged the changes

**Comment by kheos31** [comment]
Big thanks @pavankjadda ! 🙏 ❤️
